ext_localconf.php

```diff
@@ -13,15 +13,12 @@
 );
 
 if (TYPO3_MODE === 'BE') {
```

Comment:
This comparision should not be used anymore in v10, so lets remove it too

Reply:
Is there an alternative? Or is it no longer needed?

Reply:
Not needed

Reply:
Ok, fixed